srcStatic/GameModules/IniModule.cpp

```diff
@@ -40,3 +41,15 @@ bool IniModule::Unload()
 
 	return success;
 }
+
+std::string IniModule::Directory()
+{
+	auto dllSetting = DllName();
+	return fs::path(dllSetting).remove_filename().string();
```

Comment:
If the DLL name includes the path, that will be enough to distinguish the module name.

In terms of the default name, we are sort of stuck since console modules are always named "op2mod.dll". By setting a default for INI modules that matches, we are able to load console modules with minimal INI setup. In particular, most console modules can then be loaded as INI modules, without needing an explicit INI section to specify the DLL name.

Reply:
Good point on console modules. Maybe we can visit allowing other dll filenames for console modules in an issue.

Reply:
Actually, you may have a good point there. This seems related to an idea I had about untying resource loading from module loading.